RecoLocalMuon/GEMSegment/plugins/ME0SegFit.cc

```diff
@@ -0,0 +1,515 @@
+// ------------------------- //
+// --> ME0SegFit.cc 
+// Created:  11.05.2015
+// --> Based on CSCSegFit.cc
+// with last mod: 03.02.2015
+// as found in 750pre2 rel
```

Comment:
it was more to keep track at which point in history of CSCSegFit.cc this code was taken as a basis of ME0SegFit. I removed it.